### DISCUSSION TITLE
INFO / package => TigerVNC 1.14 => Xvnc HW accelerated OpenGL and Vulkan, etc.

### DISCUSSION THREAD

**Discussion by JOs86** [termux/termux-packages#21919] [Ideas]
TigerVNC 1.14.0
Xvnc now supports hardware accelerated OpenGL and Vulkan
https://github.com/tigervnc/tigervnc/releases

Any news/insights concerning the package version in Termux?

**Comment by licy183** [comment]
> Xvnc now supports hardware accelerated OpenGL and Vulkan on drivers that supports GBM

But sadly, due to the limitation of Android system, GBM is not supported by Termux.

**Comment by jcharth** [comment]
i did not know vnc worked on termux. that is nice. i use phone link and air drodi. at some point i compiled tightvnc with stunnel for encrypted communication but that was a long time ago. for now i am trying to install opencv python on android 6
